pkg/util/util.go

```diff
@@ -92,22 +93,29 @@ func MapToKeyValuePairs(m map[string]string) string {
 
 func DetectOpenShift() (isOpenshift bool, isOpenshift4 bool, anError error) {
 	tests := IsTestMode()
-	if !tests {
-		apiGroups, err := getApiList()
-		if err != nil {
-			return false, false, err
+	if tests {
+		openshiftVersionEnv := os.Getenv("OPENSHIFT_VERSION")
+		openshiftVersion, err := strconv.ParseInt(openshiftVersionEnv, 0, 64)
```

Comment:
I would add try/catch here to rely of apiGroup if parsing failed.

Reply:
If something went wrong we set up Openshift 3. Because it was earlier behavior.

Reply:
Forget this, my confusion.